base/src/main/proto/spine/options.proto

```diff
@@ -548,7 +553,13 @@ message IfMissingOption {
     option (default_message) = "A value must be set.";
 
     // A user-defined validation error format message.
-    string msg_format = 1;
+    //
+    // Use `error_msg` instead.
+    //
+    string msg_format = 1 [deprecated = true];
+
+    // A user-defined error message.
+    string error_msg = 2;
```

Comment:
Please say something about the available placeholders (i.e. `other`, `value`) which you mention in the example below.

Reply:
We mention the placeholders where it makes sense. There is, however, no reason to add those tokens to `IfInvalid` and `IfMissing` error messages.